examples/src/main/java/com/jsyn/examples/ListAudioDevices.java

```diff
@@ -34,10 +38,10 @@ public static void main(String[] args) {
             int maxOutputs = audioManager.getMaxInputChannels(i);
             boolean isDefaultInput = (i == audioManager.getDefaultInputDeviceID());
             boolean isDefaultOutput = (i == audioManager.getDefaultOutputDeviceID());
-            System.out.println("#" + i + " : " + deviceName);
-            System.out.println("  max inputs : " + maxInputs
+            LOGGER.debug("#" + i + " : " + deviceName);
```

Comment:
This program is supposed to list devices to the Terminal. Does it still do that?

Reply:
Yes, functionality remains unchanged other than adding the timestamp to the message via the logging implementation (I double-checked this as well)

Reply:
Adding a timestamp to each line would be really annoying. Imagine entering 'df' in Linux and seeing a timestamp on every line. These are not logs. They are the output of a command line program.